app/src/main/java/ch/epfl/sdp/entity/AoeRadiusMovingEntity.java

```diff
@@ -0,0 +1,26 @@
+package ch.epfl.sdp.entity;
+
+/**
+ * Represents a moving entity that has AOE radius.
+ */
+public class AoeRadiusMovingEntity extends MovingEntity {
+    private double aoeRadius;
+
+    public double getAoeRadius() {
+        return this.aoeRadius;
+    }
+
+    public void setAoeRadius(double aoeRadius) {
+        this.aoeRadius = aoeRadius;
+    }
+
+    @Override
+    public EntityType getEntityType() {
+        return null;

```

Comment:
Maybe NONE or something else? But normally we should not need entityType soon (we will change the way we display entities)

Reply:
its a temporary refactoring